datacommons/key.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2022 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" API key related functions.
+"""
+
+import os
+
+# Environment variable for API key.
+_KEY_ENV = 'DC_API_KEY'
```

Comment:
Does the user need to set it?  Should we talk about it in README/docs?

Reply:
Yes, the user will need to set it. We should add this back in docs.